config/initializers/doorkeeper.rb

```diff
@@ -58,6 +60,15 @@
   #     redirect_to sign_in_url
   #   end
   # end
+  #
+  # WARNING, the following code allow any user to edit Client Application endpoints!
+  admin_authenticator do
+    if current_user
+      head :forbidden unless current_user.id == 1
```

Comment:
I guess adding boolean to the user should be better right? Comparing `id == 1` feels so weird. Altough this security practice of restricting user access to doorkeeper panel is not required in the IC, but if we add it, the implementation should be ok. So we can either remove it or add it the proper way :)

Reply:
"if we add it, the implementation should be ok" ==> Well said! I like that! :) 

Done ✅